### ISSUE TITLE
Accuracy Measures for Neural Networks in Readme

### ISSUE THREAD

**Issue by dlshriver** [eth-sri/eran#30]
Hello,
Are the accuracies of the neural networks that are included in the README ([here](https://github.com/eth-sri/eran#neural-networks-and-datasets)) reported anywhere? I'm trying to load the networks using custom tooling and I just want to check that I am maintaining the original performance.

**Comment by GgnDpSngh** [comment]
Hi there,

Thanks for your interest in using the networks. It should be possible to obtain the required accuracies by running ERAN with epsilon=0 otherwise I can also provide accuracies for the networks you intend to use.

Cheers,
Gagandeep Singh

**Comment by dlshriver** [comment]
Thank you. Does running with epsilon=0 give results on the full test set? I am primarily interested in the test accuracy for the CIFAR resnet network. Do you know what it should be?

**Comment by GgnDpSngh** [comment]
Yes, it can be found in Table 3 here:

https://arxiv.org/pdf/1903.12519.pdf

Note that other ResNets from that Table are available in the PyTorch format here:
https://www.dropbox.com/sh/66obogmvih79e3k/AACe-tkKGvIK0Z--2tk2alZaa?dl=0

Cheers,
Gagandeep Singh

**Comment by dlshriver** [comment]
Thank you! That helps a lot.
